data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -3,6 +3,8 @@ set -e
 
 mkdir --parents /etc/kubernetes/{manifests,bootstrap-configs,bootstrap-manifests}
 
+export REGISTRY_AUTH_FILE=/opt/tectonic/pull-auth.json
```

Comment:
Rather than exporting env variable, can we just put this secret to a location that adds it to the deafult for all `root` users. Users might be debugging on bootstrap node so it will be easier if we don't have to document another step reagrding setting this env when trying to pull protected content...

Reply:
Also the kubelet needs it to pull images for static pods. Ref https://kubernetes.io/docs/concepts/containers/images/#configuring-nodes-to-authenticate-to-a-private-registry

Reply:
> Rather than exporting env variable, can we just put this secret to a location that adds it to the deafult for all root users.

Done with e8fe290 -> 782da57.